### ISSUE TITLE
deno "std/log" not working 

### ISSUE THREAD

**Issue by ceo** [denoland/deno#5643]
I was reading/trying this with v1.0 :

https://deno.land/std/log

Tried this code :

```
import * as log from "https://deno.land/std/log/mod.ts";

// Simple default logger out of the box. You can customize it
// by overriding logger and handler named "default", or providing
// additional logger configurations
log.debug("Hello world");
log.info("Hello world");
log.warning("Hello world");
log.error("Hello world");
log.critical("500 Internal server error");
```

Expected behaviour: 
Logs should be printed on terminal.

Current behaviour:
➜  ~ deno run fs.ts
Compile file:///Users/macuser/fs.ts
➜  ~ 

No logs showing..  any idea why ?

Thanks

**Comment by cknight** [comment]
This will be fixed once https://github.com/denoland/deno/pull/5341 lands.  In the meantime, this code will setup a basic console logger for you:
```ts
await log.setup({
  handlers: {
    console: new log.handlers.ConsoleHandler("DEBUG"),
  },

  loggers: {
    default: {
      level: "DEBUG",
      handlers: ["console"],
    },
  },
});

const logger = log.getLogger();
// These will log to console
logger.debug("fizz");
logger.warning("buzz");
```

**Comment by matthieudesprez** [comment]
Actually, just call the setup with an empty config and an await does the trick
```typescript
import * as log from "https://deno.land/std/log/mod.ts";

await log.setup({});
log.debug("Hello world");
log.info("Hello world");
log.warning("Hello world");
log.error("Hello world");
log.critical("500 Internal server error");
```

**Comment by bartlomieju** [comment]
Fixed in #5341 
Will be released in `std/0.52.0`
